src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.domain
+
+data class WinningLotto(
+    val winningTicket: LottoTicket,
+    val bonusNumber: LottoNumber
+) {
+    fun matching(lottoTicket: LottoTicket): LottoRank {
+        return LottoRank.of(
+            lottoTicket.matching(winningTicket),
+            lottoTicket.contains(bonusNumber)
+        )
```

Comment:
of 함수의 시그니처를 보지 않으면 이 두 arguments가 어떤 역할을 하는지 호출부에서 구분하기 어렵습니다. 그래서 전 두 개 이상의 arguments를 넘길 때에는 named arguments를 활용하여 arguments에 이름을 주는걸 선호합니다.

Effective Kotlin의 [[Item 17 - Consider naming arguments]](https://dalsacoo-log.tistory.com/entry/Effective-Kotlin-Item-17-Consider-naming-arguments?category=907839?category=907839)를 참고하시면 도움이 될거예요!


```suggestion
        return LottoRank.of(
            matchCount = lottoTicket.matching(winningTicket),
            bonusMatch = lottoTicket.contains(bonusNumber)
        )
```

FYI: [shortcut](https://www.jetbrains.com/idea/guide/tips/adding-call-args/)을 사용하시면 훨씬 편합니다!